common/djangoapps/student/views.py

```diff
@@ -1508,6 +1508,8 @@ def _do_create_account(form, custom_form=None):
     user = User(
         username=form.cleaned_data["username"],
         email=form.cleaned_data["email"],
+        first_name=form.cleaned_data.get("first_name", ""),
+        last_name=form.cleaned_data.get("last_name", ""),
```

Comment:
We shouldn't need to add these here. These fields will be persisted in auth_userprofile.meta.